### PR TITLE
Update DG Appendix test

### PR DESCRIPTION
**Pull request by wesho1107**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1687,612 +1687,695 @@ testers are expected to do more *exploratory* testing.
...
+    1. There is at least 1 student being displayed in the students list.
+  2. Test case: `editgradedtest 1 ra1/-90 ra2/-85 mt/-100 f/invalid pe/-75 ra1/-88`
+     Expected: F.A.K.E.J.A.R.V.I.S. displays an error. Graded Test scores are not edited. <br> Reason: The given scores are invalid.
 
 ### Consultation
 
 #### Creating a consultation
 
-1. Creating a consultation with specified date, time and student(s).
+1. Creating a consultation with a single student and _valid_ parameters.
```
Comment (ldinghan) [inline docs/DeveloperGuide.md:1861]: should be \_valid parameters_ instead, same for below

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1687,612 +1687,695 @@ testers are expected to do more *exploratory* testing.
...
+2. Creating a consultation with multiple students and _valid_ parameters.
+   1. Prerequisites:
+      1. Date and time inputs must be valid
+      2. Student names must exist in student list.
+      3. There must not be a duplicate consultation (same date, time and students).
+   
+   2. Test case: `createconsult d/10/11/2023 tt/13:00 n/Alex Yeoh n/Bernice Yu`<br>
+      Expected: A consultation on 10/11/2023 at 13:00 with student Alex Yeoh and Bernice Yu, is created and stored in the list of consultations.
+
+3. Creating a consultation with multiple students and _invalid_ parameters.
```
Comment (ldinghan) [inline docs/DeveloperGuide.md:1879]: This one as well, \_invalid parameters_

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1687,612 +1687,695 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `createconsult d/10/11/2023 tt/13:00 n/Alex Yeoh` `createconsult d/10/11/2023 tt/13:00 n/Alex Yeoh`<br>
+      Expected: F.A.K.E.J.A.R.V.I.S. displays an error. No new consultation is created on the second command.<br>
+      Reason: There must not be duplicated consultations with same date, time and students when creating.
```
Comment (ldinghan) [inline docs/DeveloperGuide.md:1882]: Incomplete sentence or some grammatical thing line 1882

**Review by Respirayson — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -1687,612 +1687,695 @@ testers are expected to do more *exploratory* testing.
...
+      Reason: Date parameter is missing.
+   2. Test case: `createconsult d/10/11/2023 n/Alex Yeoh`.<br>
+      Expected: F.A.K.E.J.A.R.V.I.S. displays an error. No new consultation is created.<br>
+      Reason: Time parameter is missing.
+   3. Test case: `createconsult d/10/11/2023 tt/13:00`.<br>
+      Expected: F.A.K.E.J.A.R.V.I.S. displays an error. No new consultation is created.<br>
+      Reason: Student names parameter is missing.
+   4. Test case: `createconsult`.<br>
+      Expected: F.A.K.E.J.A.R.V.I.S. displays an error. No new consultation is created.<br>
+      Reason: There are no parameters inputted.
```
Comment (ldinghan) [inline docs/DeveloperGuide.md:1910]: "Both parameters are missing" line 1910

**Pull request merged**